app/Http/Controllers/User/HomeController.php

```diff
@@ -14,27 +14,27 @@ class HomeController extends Controller
 {
     public function index(Request $request)
     {
-    	$request = $request->all();
-    	$teachers = Teacher::orderBy('id','ASC');
-    	if (isset($request['grades'])) {
-    		$teacherIds = [];
-    		foreach ($teachers as $key => $teacher) {
-    			$grade = array_intersect($request['grades'], $teacher->grade);
-    			if (!empty($grade)) {
-    				$teacherIds[] = $teacher->id;
-    			}
-    			$teachers = $teachers->whereIn('id', $teacherIds)->get();
-    		}
-    	}
+        $request = $request->all();
+        $teachers = Teacher::orderBy('id','ASC');
```

Comment:
**Reporter**: CHECKSTYLE</br>**Rule**: Generic.Functions.FunctionCallArgumentSpacing.NoSpaceAfterComma</br>**Severity**: error</br>**File**: app/Http/Controllers/User/HomeController.php</br>**Line**: 18</br>No space found after comma in function call